app/assets/stylesheets/components/_filter-panel.scss

```diff
@@ -14,7 +14,7 @@
   align-items: center;
   gap: govuk-spacing(2);
 
-  .gem-c-heading {
+  .js-app-c-filter-panel__count {
```

Comment:
The `js-` prefix is only for elements that are used as a target in Javascript. Don't think this is? Should probably just be `.app-c-filter-panel__count`.

Reply:
Maybe instead of using the heading component, we can just manually add a heading in given that we're overriding the styling anyway?